lib/devices/gen2-helper.js

```diff
@@ -115,6 +115,22 @@ function addEM1(deviceObj, switchId) {
             unit: 'Hz',
         },
     };
+deviceObj[`EM1:${switchId}.TotalEnergy`] = {
+        device_mode: 'em1',
+        mqtt: {
+            mqtt_publish: `<mqttprefix>/status/em1data:${switchId}`,
+            mqtt_publish_funct: value => JSON.parse(value).total_act_energy,
+        },
+        common: {
+            name: 'Total Energy',
+            type: 'number',
+            role: 'value.total_energy',
```

Comment:
That role doesn't exist?!

Reply:
role changed to existing role value.energy